.github/workflows/test.yml

```diff
@@ -417,3 +417,28 @@ jobs:
           make test
           ./test
         working-directory: ./stdlib
+
+  coverage:
+    runs-on: ubuntu-latest
+    container: ghcr.io/hyperledger/solang-llvm:ci-5
```

Comment:
Meanwhile we bumped this to `ci-6`, don't forget to update the image before merging tag

Reply:
That has not been merged.